cmd/agent/subcommands/flare/command.go

```diff
@@ -278,6 +280,20 @@ func makeFlare(flareComp flare.Component,
 		err     error
 	)
 
+	streamLogParams := streamlogs.CliParams{
+		FilePath: commonpath.DefaultStreamlogsLogFile,
+		Duration: cliParams.streamLogsDuration,
+		Quiet:    true,
+	}
+
+	// Set default duration to 60 seconds if not provided by the user
+	const defaultStreamLogDuration = 60 * time.Second
+	if streamLogParams.Duration <= 0 {
+		streamLogParams.Duration = defaultStreamLogDuration
+	}
+	// If Duration is set, automatically enable withStreamLogs.
+	cliParams.withStreamLogs = streamLogParams.Duration > 0
```

Comment:
Hmm - I'm not sure what's the purpose of `--with-stream-logs` flags if duration effectively overrides it in all cases. Maybe we want to use "number of seconds" as the flag and not duration [similar to the flare command](https://github.com/DataDog/datadog-agent/blob/main/cmd/agent/subcommands/flare/command.go#L150)?

Reply:
Also, because default is `0`, this logic always sets `streamLogParams.Duration = 60 * time.Second` and `cliParams.withStreamLogs = true` if neither flag is set (which isn't correct afaiu)